app/pages/cif/projects.tsx

```diff
@@ -113,7 +113,12 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
         allProjectStatuses.edges.map((e) => e.node.name),
         { orderByPrefix: "PROJECT_STATUS_BY_PROJECT_STATUS_ID__NAME" }
       ),
-      new SortOnlyFilter("Milestone Due", "nextMilestoneDueDate"),
+      new SearchableDropdownFilter(
+        "Milestone Due",
+        "milestoneStatus",
+        ["Complete", "None", "On track", "Late"],
```

Comment:
I considered putting the statuses into a table like `project_status`, but the forms handle report status differently, so the table didn't seem very resuable